apps/ui/vite.config.ts

```diff
@@ -15,6 +15,9 @@ const target = ['esnext'];
 
 export default defineConfig({
   base: ELECTRON ? path.resolve(__dirname, './dist') : undefined,
+  server: {
+    host: '127.0.0.1'
+  },
```

Comment:
we will not have localhost here? 
<img width="408" alt="image" src="https://github.com/user-attachments/assets/a0b056d1-acd8-4c09-a121-3dc95c18ee31">